cache/src/main/scala/coursier/TermDisplay.scala

```diff
@@ -509,4 +509,9 @@ class TermDisplay(
     }
   }
 
+  override def log(content: String, currentTimeOpt: Option[Long]): Unit = {
+    updateRunnable.newEntry(content, CheckUpdateInfo(currentTimeOpt, None, isDone = true), content)
+    updateRunnable.removeEntry(content, success = true, fallbackMessage = content)(x => x)
```

Comment:
Not entirely sure if this is the right way to log things. Please kindly advise.

Reply:
Let's keep an empty implem for now (with a more specific method), we'll see how we can report that (with progress bars and all) later…